tutorial/TUTORIAL.md

```diff
@@ -92,8 +93,10 @@ Now we can export our program using "File->Export" and saving it as "web.py"
 (make sure it is saved in the same location where you stored and executed
 'gendb.py'). Afterwards we can run it:
 ```
-    python2.7 web.py > web.html
+    python2 web.py > web.html
 ```
+Note that we need Python 2 here as Eco still generated Python 2 files.
```

Comment:
Maybe: "Note that Eco currently still generates Python2 code which is why we need to use `python2` to run it."